fix: testnet archival node

spypsy opened the pull request:
- Fixes issue with config where '10000000' config was converted to '1e+07'
- Adds archival node deployment to new helm charts

spypsy commented on spartan/terraform/deploy-testnet/values/testnet-archive-node.yaml:
```diff
@@ -28,7 +28,7 @@ affinity:
 node:
   env:
-    P2P_ARCHIVED_TX_LIMIT: 10000000 # 10 million
+    P2P_ARCHIVED_TX_LIMIT: "10000000" # 10 million
```
FYI this is the reason the previous deployment didn't work

alexghr commented on spartan/terraform/deploy-aztec-infra/values/archive-resources-prod.yaml:
```diff
@@ -0,0 +1,45 @@
+  resources:
+    requests:
+      cpu: "3"
+      memory: "12Gi"
```
These requests need to be lower otherwise the pod won't fit on a 2-core machine

```suggestion
      cpu: "1.5"
      memory: "5Gi"
```

spypsy committed:
fix: testnet archival node

- Fixes issue with config where '10000000' config was converted to '1e+07'
- Adds archival node deployment to new helm charts
